libbpfgo/selftest/perfbuffers/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+GREEN='    \033[0;32m'
+RED='    \033[0;31m'
+NC='\033[0m' 
+
+VERSION_LIMIT=5.8
+CURRENT_VERSION=$(uname -r | cut -d '.' -f1-2)
+
+# Check major version
+MAJOR_VERSION_LIMIT=$(echo $VERSION_LIMIT | cut -d '.' -f1)
+MAJOR_CURRENT_VERSION=$(echo $CURRENT_VERSION | cut -d '.' -f1)
+if (( $(echo "$MAJOR_CURRENT_VERSION < $MAJOR_VERSION_LIMIT") |bc -l)); then
+    echo -e "${RED}[*] OUTDATED MAJOR KERNEL VERSION${NC}"
+    exit 1
+fi
+
+# Check minor version
+MINOR_VERSION_LIMIT=$(echo $VERSION_LIMIT | cut -d '.' -f2)
+MINOR_CURRENT_VERSION=$(echo $CURRENT_VERSION | cut -d '.' -f2)
+if (( $(echo "$MINOR_CURRENT_VERSION < $MINOR_VERSION_LIMIT") |bc -l)); then
+    echo -e "${RED}[*] OUTDATED MINOR KERNEL VERSION${NC}"
```

Comment:
if 
$MAJOR_CURRENT_VERSION > $MAJOR_VERSION_LIMIT - 
there is no reason to check for minor, otherwise, version like 6.0 will not pass this test, right?